src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+$--main-color: #fff;
+$--code-color: #616070;
+$--button-color: #00acdc;
+$--text-color: #060b35;
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)